tsdb/block_test.go

```diff
@@ -303,7 +304,13 @@ func TestReadIndexFormatV1(t *testing.T) {
 
 // createBlock creates a block with given set of series and returns its dir.
 func createBlock(tb testing.TB, dir string, series []Series) string {
-	return createBlockFromHead(tb, dir, createHead(tb, series))
+	w, close := newTestNoopWal(tb)
```

Comment:
If it's noop and also test WAL, why even need to close it then? ;p

Reply:
`close` is for cleaning up the temporary dir used for the WAL. WAL is noop, but the chunks will be written to the disk by the Head in the mmap PR :)